### ISSUE TITLE
Backend yarn start: unexpected error occurred: "Command "start" not found."

### ISSUE THREAD

**Issue by willpmartin** [postlight/headless-wp-starter#68]
getting an error when using the above and getting the following error

yarn install && yarn start
yarn install v1.5.1
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.70s.
yarn run v1.5.1
error An unexpected error occurred: "Command \"start\" not found.".
info If you think this is a bug, please open a bug report with the information provided in "/Users/willmartin/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Ive attached the error log for more details. The package.json file looks to be correct but not sure what the issue is 

Thanks for any help and for building an awesome tool! 

[yarn-error.log](https://github.com/postlight/headless-wp-starter/files/1904208/yarn-error.log)

**Comment by jrdubina** [comment]
Does your package.json have "start": "wp server" in the scripts section?

**Comment by willpmartin** [comment]
Thanks for the reply. 

Here is the contents of the package.json

{
  "name": "headless-wp-starter",
  "version": "1.0.0",
  "repository": "git@github.com:postlight/headless-wp-starter.git",
  "author": "Postlight",
  "license": "UNLICENSED",
  "private": true,
  "scripts": {
    "preinstall": "chmod +x install.sh && ./install.sh",
    "postinstall": "robo wordpress:setup",
    "start": "wp server"
  }
}

**Comment by willpmartin** [comment]
other package.json file

{
    "name": "frontend",
    "version": "1.0.0",
    "description": "",
    "main": "index.js",
    "scripts": {
        "dev": "node server.js",
        "build": "next build",
        "start": "next start",
        "start": "wp server",
        "docker:build": "docker build -t frontend .",
        "docker:clean": "docker rm -f frontend || true",
        "docker:run": "docker run -p 3000:3000 --name frontend frontend",
        "docker:stop": "docker stop frontend",
        "docker:start": "docker start frontend && yarn run docker:logs",
        "docker:logs": "docker logs -f frontend",
        "deploy":
            "yarn run docker:build && yarn run docker:clean && yarn run docker:run"
    },
    "keywords": [],
    "author": "",
    "license": "ISC",
    "dependencies": {
        "express": "^4.16.2",
        "isomorphic-unfetch": "^2.0.0",
        "next": "latest",
        "react": "^16.0.0"
    },
    "devDependencies": {
        "autoprefixer": "7.1.5",
        "babel-plugin-module-resolver": "^2.7.1",
        "babel-plugin-wrap-in-js": "^1.1.0",
        "glob": "^7.1.2",
        "node-sass": "^4.4.0",
        "normalize.css": "^7.0.0",
        "postcss-easy-import": "^3.0.0",
        "postcss-loader": "^2.0.7",
        "raw-loader": "^0.5.1",
        "react-dom": "^16.2.0",
        "sass-loader": "^6.0.6",
        "webpack": "^3.10.0"
    }
}

**Comment by krichdev** [comment]
@willpmartin - I was having this same issue. I [un-installed homebrew](https://github.com/Homebrew/brew/blob/master/docs/FAQ.md#how-do-i-uninstall-homebrew) then [reinstalled homebrew](https://brew.sh/), then [reinstalled yarn](https://yarnpkg.com/lang/en/docs/install/#mac-stable) and re-ran the `yarn start` command and it worked.

**Comment by willpmartin** [comment]
@krichdev Thanks for the reply. 

I update yarn to v 1.6.0 

Still getting the same error 

yarn install && yarn start
yarn install v1.6.0
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.77s.
yarn run v1.6.0
error Command "start" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by willpmartin** [comment]
Sorted now, started over and everything seemed to work ok! thanks @krichdev @ginatrapani @jrdubina for your help!

**Comment by RajneeshShukla** [comment]
try npm start. 
[](https://github.com/react-community/create-react-native-app)

Create React Native App has been merged with Expo CLI You can now use expo init to create your project

**Comment by ttstauss** [comment]
Following the documentation I was finally able to get to the following message in my console:

> [OK] Great. You can now log into WordPress at: http://localhost:8080/wp-admin (nedstark/winteriscoming)

However, when I go to the given url, I get a "This site can’t be reached localhost refused to connect." error message.

After some investigation, it seems like the documentation is missing the step to actually start the WordPress server (I do see see the _wp-headless_ and _db-headless_ containers in Docker.). Is that correct?

Following the discussion above, how does the following commit change the proposed solutions (where "start": "wp server" was removed from the root package.json file)?

https://github.com/postlight/headless-wp-starter/commit/d2fccc7d3305842b65163407b12876b242a79917#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

I tried adding back "start": "wp server" to the root package.json file, but I get a new error:

```
Error: YIKES! It looks like you're running this as root. You probably meant to run
this as the user that your WordPress installation exists under.

If you REALLY mean to run this as root, we won't stop you, but just bear in mind
that any code on this site will then have full control of your server, making it quite DANGEROUS.

If you'd like to continue as root, please run this again, adding this flag:  --allow-root

If you'd like to run it as the user that this site is under, you can run the following
to become the respective user:
    sudo -u USER -i -- wp <command>
```

I'm a little new to this, so I apologize if I'm missing something obvious here. I can't seem to figure it out though. Let me know if I need to open a new issue too, since this one is technically closed.

Any help would be greatly appreciated.

Thanks!
